server/config.ts

```diff
@@ -111,6 +111,15 @@ export default {
       },
       agent: new AgentConfig(Number(get('HMPPS_PRISON_API_TIMEOUT_RESPONSE', 10000))),
     },
+    offenderSearchApi: {
+      url: get('OFFENDER_SEARCH_API_URL', 'http://localhost:8082', requiredInProduction),
```

Comment:
port 8082 is already in use by another api so you can go for 8083… but in practice, none of us runs this site locally against services in docker so it doesn't really matter.

this fallback settings is _supposed_ to be for running the site along with everything else it needs in docker compose. we've never managed to do that cos of auth issues so we all use services in `dev` instead by actually setting the environment variables. so really, this fallback setting isn't necessary – tho for consistency, might as well have 8083 i guess

Reply:
That would make sense why it wasn't flagging up as already used. Thank you for catching that - missed that port on frontendComponents.